geth/jail/jail.go

```diff
@@ -80,18 +80,27 @@ func (j *Jail) Stop() {
 	j.cells = make(map[string]*Cell)
 }
 
-// createCell creates a new cell if it does not exists.
-func (j *Jail) createCell(chatID string) (*Cell, error) {
+// obtainCell returns an existing cell for given ID or
+// creates a new one if it does not exist.
+// Passing in true as a second argument will cause a non-nil error if the
+// cell already exists.
+func (j *Jail) obtainCell(chatID string, expectNew bool) (cell *Cell, err error) {
```

Comment:
How about keeping `obtainCell()` as an internal function, but adding two more user-friendly wrappers for both `expectNew` possible values — namely `createCell` and `getCell`.

That would change code:

```go
j.obtainCell(chatID, true)
j.obtainCell(chatID, false)
```
to 
```go
j.createCell(chatID)
j.getCell(chatID)
```
which is more readable, decrease probability of making a mistake, doesn't require introspecting parameter names and meanings.

So essentially I see it as simple as adding two more functions:
```go
func (j *Jail) createCell(chatID string, expectNew bool) (cell *Cell, err error) {
    return j.obtainCell(chatID, true)
}

func (j *Jail) getCell(chatID string, expectNew bool) (cell *Cell, err error) {
    return j.obtainCell(chatID, false)
}
```

Reply:
It's a good comment but the PR is already merged, just do another one :)